.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,36 @@
+name: Chorus .NET6
+
+on:
+  [push, pull_request, workflow_dispatch]
+
+jobs:
+  build_test_deploy:
+    env:
+      NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    runs-on: windows-latest
+    permissions: write-all
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # fetch full history for GitVersion
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+        source-url: https://nuget.pkg.github.com/sillsdev/index.json
+    - name: Restore
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore -c Release
+    - name: Test
+      run: dotnet test
+    - name: Pack
+      if: github.ref == 'refs/heads/develop'
```

Comment:
Shouldn't this be `refs/heads/master`? (also below)

Reply:
good catch, will fix